paperqa/sources/clinical_trials.py

```diff
@@ -29,45 +29,58 @@
 SEARCH_PAGE_SIZE = 1000
 TRIAL_API_FIELDS = "protocolSection,derivedSection"
 DOWNLOAD_CONCURRENCY = 20
-TRIAL_CHAR_TRUNCATION_SIZE = 30_000  # larger will prevent embeddings from working
+TRIAL_CHAR_TRUNCATION_SIZE = 28_000  # larger will prevent embeddings from working
 MALFORMATTED_QUERY_STATUS: int = 400
 
 
+class CookieWarningFilter(logging.Filter):
+    """Filters out bad cookie warning due to clincialtrials.gov response."""
```

Comment:
Can you update to state:

- Why the warning is "bad"
- What is "bad" exactly

Maybe reword to say """Filter out log spam from to clincialtrials.gov responses."""